SwiftWisdom/Core/Foundation/Data/DataConvertible.swift

```diff
@@ -18,13 +18,24 @@ public protocol DataConvertible {
 extension DataConvertible {
     
     public init?(ip_data data: Data) {
-        guard data.count == MemoryLayout<Self>.size else { return nil }
-        self = data.withUnsafeBytes { $0.pointee }
+        let size = MemoryLayout<Self>.size
+        if data.count < size {
+            let difference = size - data.count
+            var modifiedData = data
+            for _ in 0..<difference {
+                modifiedData.append(0)
```

Comment:
Does anyone know if appending an array of size `difference` would be faster than iterating over `difference` and appending one by one?

`modifiedData.append(contentsOf: Array<UInt8>(count:difference, repeatedValue: 0))`